modules/proc_deploy-quay-openshift-operator.adoc

```diff
@@ -885,6 +966,48 @@ spec:
     imagePullSecretName: redhat-pull-secret
 ```
 
+The Quay Operator sets the Clair database connection string with the parameter
+`sslmode=disable` if no parameters are specified in QuayEcosystem custom
+resource. In case you have SSL enabled Postgres database, or want to add
+other parameters, provide `key: value` pairs as strings (for example,
+ connect_timeout: '10') under connectionParameters object.
+
+For example: 
+
+```
+apiVersion: redhatcop.redhat.io/v1alpha1
+kind: QuayEcosystem
+metadata:
+  name: example-quayecosystem
+spec:
+  quay:
+    imagePullSecretName: redhat-pull-secret
+  clair:
+    enabled: true
+    imagePullSecretName: redhat-pull-secret
+    database:
+      connectionParameters:
+        sslmode: require
+        connect_timeout: '10'
+```
+
+Supported connection string parameters:
+
+* **sslmode** - Whether or not to use SSL (default is require, this is not the default for libpq)
```

Comment:
default value of `sslmode` should be disable

Reply:
Need more information here.

Reply:
Never mind. I got it now. Done.